core/domain/config_domain.py

```diff
@@ -328,7 +359,7 @@ def init_config_property(cls, name, instance):
         cls._config_registry[name] = instance
 
     @classmethod
-    def get_config_property(cls, name):
+    def get_config_property(cls, name: str) -> ConfigProperty:
```

Comment:
Looking at the code more carefully the return value should be `Optional[ConfigPropery]`

Reply:
Done!